opencensus/proto/agent/trace/v1/trace_service.proto

```diff
@@ -32,16 +32,16 @@ option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/a
 service TraceService {
   // After initialization, this RPC must be kept alive for the
   // entire life of the application. The agent pushes configs
-  // down to applications via a stream of requests.
-  rpc Config(stream ConfigTraceServiceResponse) returns (stream ConfigTraceServiceRequest) {}
+  // down to applications via a stream.
+  rpc Config(stream RequestedUpdatedConfig) returns (stream CurrentLibraryConfig) {}
```

Comment:
Nice suggestions! How about changing from 
`RequestedUpdatedConfig`
to
`LibraryConfigUpdate`

but we need to preserve the order of the streams though to
```proto
rpc Config(stream CurrentLibraryConfig) returns (stream LibraryConfigUpdate) {}
```
lest we go back to the scenario yesterday.

Reply:
Oh yes, I got the order of streams wrong. The name "LibraryConfigUpdate" sounds better to me.